### DISCUSSION TITLE
Draft repo

### DISCUSSION THREAD

**Discussion by crazymonkyyy** [dlang/PhobosV3-Design#16] [Ideas]
I have code; take some.  While I'm unwilling to tolerate a slow bureaucratic process with a very high failure rate Id like to put some code somewhere to test the waters on considering larger contributions and you should be seeing how ideas interact with temp code.

You'll need to have the spine to delete 1/2 of the lines of code of course, but you should have a draft repo.

**Comment by LightBender** [comment]
I've considered opening one right here. But I'd like to get a resolution to this first so that we can not have to completely rerog things later: https://github.com/LightBender/PhobosV3-Design/discussions/9

I am open to removing things, and many people have stated that they would like to see things removed, however it appears that nobody really agrees on what should be removed, and we should also be mindful that the things in the StdLib are there for a reason and somebody probably uses it, even if they aren't active in the Forums. You can post your list and we can debate it?

**Reply by crazymonkyyy** [reply]
> I am open to removing things

I meant from the draft repo, permissive, let people upload whatever to see how peoples ideas clash, then for an alpha you purge.

My ideal is closer to adr's plan, I want first party libs and no mandated dub; besides the awful naming conflicts of std.file and std.stdio and making some sane quality of life renames, Id leave std as is.

If you havnt heard my rant on the subject, I dont use dub, I hate dub, I believe community interfaces like the range concept are the only way to have generic code and there needs to be two tiers of projects, std/first party libs and utility files you copy and paste and are 100% responsible for mantaining and understanding.

**Comment by pbackus** [comment]
You can always post your code in its own repo and share a link to it. That's what I'm doing with my `@safe` allocators proof-of-concept.

**Comment by LightBender** [comment]
Now that development is open, this is a moot point. Please feel free to submit PR's to the main Phobos repo!
